scripts/jest/jest-cli.js

```diff
@@ -258,7 +258,7 @@ function getCommandArgs() {
 
   // Push the remaining args onto the command.
   // This will send args like `--watch` to Jest.
-  args.push(argv._);
+  args.push(...argv._);
```

Comment:
(`argv._` is an array of additional args.)

Reply:
Whoops nice catch